### PR TITLE
Merges codespaces-service components into cascade-agent-cli

### PR DESCRIPTION
**Pull request by anpaskin**
Moves libssl install to dockerfile, configures testsettings.json, removes duplicate dotnet install from post-create

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,45 @@
...
+    * Value: Paste in the generated PAT
+    * Repository access: `vsls-contrib/vscs-in-codespaces` (or if you are using a fork, select your fork)
+1. You may also add secrets to fill in certain values in the Codespace's appsettings.json, which is used for the devstamp when running the frontend and backend services.
+    * Add a secret with name `TUNNEL_KEY` and enter the value of "tunnelRelayPrimaryAuthKey" from your local appsettings.json to automatically use it in your Codespace's appsettings.json
+    * Add a secret with name `APP_SECRET` and enter the value of "appServicePrincipalClientSecret" from your local appsettings.json to automatically use it in your Codespace's appsettings.json
+    * Add a secret with name `DEVELOPER_ALIAS` and enter the value of "developerAlias" from your local appsettings.json to automatically use it in your Codespace's appsettings.json
+        * If you don't add a `DEVELOPER_ALIAS` secret, your git username will be used by default, however this will cause an error with your relay tunnel if your git username does not match the alias you used in your relay tunnel hybrid URL
```
Comment (joshspicer) [inline README.md:21]: I don't think the config I have uses the git username by default for `developer_alias`.

**Review by joshspicer — APPROVED** [review]
awesome!

**Review by joshspicer — CHANGES_REQUESTED** [review]
Just tried running the services with these changes and it looks like my appsettings.json was not automatically created?
image

File: README.md (hunk 1 above)
Comment (anpaskin) [reply README.md:21]: Good call, added that to post-create.sh

**Pull request merged**